site/static/examples/config-with-mounts.yaml

```
@@ -5,4 +5,14 @@ nodes:
   # add a mount from /path/to/my/files on the host to /files on the node
   extraMounts:
   - hostPath: /path/to/my/files/
-    containerPath: /files
\ No newline at end of file
+    containerPath: /files
+    # optional: if set, the mount is read-only.
+    # false is the current default
```

does this reads better this way?
```suggestion
    # default false
```

---

Update done in last commit